pkg/reconciler/route/traffic/rollout.go

```diff
@@ -70,3 +72,94 @@ type RevisionRollout struct {
 	// target percentage.
 	Percent int `json:"percent"`
 }
+
+// Roll processes previous Rollout object and compares to the current
+// rollout state. If there is different, Merge will start or stop the rollout
```

Comment:
What is "Merge"?

Reply:
Old name for `Roll`. At first I didn't come up with such a cool name :)